src/components/Tabs/Tab.tsx

```diff
@@ -48,6 +48,61 @@ export const useStyles = makeStyles(
         opacity: 0.4,
       },
     },
+    pill: {
+      borderBottom: 'none',
+      borderRadius: '50px',
```

Comment:
Should this use `theme.pxToRem(50)` instead to be consistent with the `borderRadius` on line 65 and the `borderRadius` set here https://github.com/lifeomic/chroma-react/pull/227/files#diff-0243aba83c9530a47511fd3f1ac4a05bc5dbfafa61918171d15d0f7a6c456994R21?

Reply:
✅